subsys/bluetooth/host/hci_core.h

```diff
@@ -156,6 +156,9 @@ enum {
 	/** Periodic Advertising Sync has been created in the host. */
 	BT_PER_ADV_SYNC_CREATED,
 
+	/** Periodic Advertising Sync Create using Advertiser List */
+	BT_PER_ADV_SYNC_USE_LIST,
+
```

Comment:
```suggestion
	/** Periodic advertising is attempting sync sync */ using Advertiser List */
	BT_PER_ADV_SYNC_SYNCING_USE_LIST,
```

And move next to the `BT_PER_ADV_SYNC_SYNCING` flag as those are the most related flags.

Reply:
Update this and other comments.